packages/auth0_fastapi/src/auth0_fastapi/auth/auth_client.py

```diff
@@ -47,37 +51,41 @@ def __init__(self, config: Auth0Config, state_store=None, transaction_store=None
                 **(config.authorization_params or {})
             },
         )
-    
-    async def start_login(self, app_state: dict = None, store_options: dict = None) -> str:
+
+    async def start_login(self, app_state: dict = None, authorization_params: dict = None, store_options: dict = None) -> str:
```

Comment:
☝️  We allow authorization_params to be provided to the `start_login` auth client method.